index.html

```diff
@@ -114,6 +114,38 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor 'ivan-qa' card START ________  -->
+        <div class="card">
+          <p class="name">Ivan</p>
+          <p class="contact">
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/ivan-krot-28324a140/" target="_blank">ivan-krot</a>
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/i_m_krot" target="_blank">@i_m_krot</a>
+          </p>
+          <p class="about">I'm an Manual QA Engineer. I like to read.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="First Resource">
+                  Stack Overflow
+                </a>
+              </li>
+              <li>
+                <a href="https://www.istqb.org/" target="_blank" title="Second Resource">
+                  ISTQB
+                </a>
+              </li>
+              <li>
+                <a href="https://dou.ua/" target="_blank" title="Third resource">
+                  DOU
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor 'ivan-qa' card END ________  -->
```

Comment:
You should also rename this comment
`*END TEMPLATE* Contributor` to your name.
This will make it easier for everyone to read the code.